app.go

```diff
@@ -5,11 +5,16 @@ import (
 	"os/exec"
 	"syscall"
 
+	"github.com/aws/aws-sdk-go/service/ssm"
 	"github.com/pkg/errors"
 )
 
 type SSMConnector interface {
 	FetchParameters(paths []string, recursive bool, retries int) (map[string]string, error)
+	FetchParametersByNames(names []string, retries int) (map[string]string, error)
```

Comment:
I think, `FetchParameters` and `FetchParametersByNames` can be redefine.

```suggestion
	FetchParametersByNames(client *ssm.SSM, names []string) (map[string]string, error)
```

is same as `fetchParametersByNames`

Reply:
I thought to keep a compatibility because FetchParameters is a public function.